### PR TITLE
LIG-5222 Allow skipping weight initialisation for Masked Vision Transformers

### PR DESCRIPTION
**Pull request by ersi-lightly**
Addresses issue #1511 and allows to skip weight initialization for  `MaskedVisionTransformerTIMM`, `MaskedVisionTransformerTorchvision` and `MAEDecoderTIMM`. This will enable to load a pretrained vit model.

### CODE REVIEW

File: lightly/models/modules/masked_autoencoder_timm.py [hunk 1]
```diff
@@ -60,6 +62,7 @@ def __init__(
         proj_drop_rate: float = 0.0,
         attn_drop_rate: float = 0.0,
         norm_layer: Callable[..., nn.Module] = partial(LayerNorm, eps=1e-6),
+        initialize_weights: bool = False,
```
Comment (guarin) [inline lightly/models/modules/masked_autoencoder_timm.py:65]:
```suggestion
        initialize_weights: bool = True,
```
I think this should be True by default, otherwise the existing tutorials/examples will break as weights will no longer be correctly initialized.

**Review by guarin — CHANGES_REQUESTED** [review]

**Review by guarin — APPROVED** [review]
LGTM!
